plugins/zenoh-plugin-trait/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2017, 2020 ADLINK Technology Inc.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+# which is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+#
+# Contributors:
+#   ADLINK zenoh team, <zenoh@adlink-labs.tech>
+#
+[package]
+name = "zenoh-plugin-trait"
+version = "0.1.0-dev"
```

Comment:
Version should be the same as zenoh (0.5.0-dev) since we will release all together.

Reply:
Good point, although I've removed `zenoh` when I realized it was causing a dependency cycle when I started working on integrating it into `zenoh` itself